src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorEnroll.ts

```diff
@@ -38,7 +38,7 @@ const getContentDescrAndParams = (brandName?: string): TitleElement['options'] =
 const isGracePeriodStillActive = (expiry: string): boolean => {
   const currentTimestampMs = Date.now();
   const gracePeriodTimestampMs = new Date(expiry).getTime();
-  return !Number.isNaN(gracePeriodTimestampMs) && currentTimestampMs < gracePeriodTimestampMs;
+  return !isNaN(gracePeriodTimestampMs) && currentTimestampMs < gracePeriodTimestampMs;
```

Comment:
This is fine and no change is needed, but the Gen3 webpack configuration should automatically add new `core-js` polyfills for the browsers we target, which includes IE11. If this works correctly, any code that is _only_ in Gen3 theoretically could use `Number.isNaN` without any issues, since it will get polyfilled at compile-time.

Reply:
Ah, no wonder I was only seeing issues with Gen 2 😅. I had applied it here also to be safe